doc/infrastructure.md

```diff
@@ -216,75 +211,78 @@ Old tracsvn (OSGeo server) (Dec 2007 * Mai 2019)
+- Rendered manuals: <https://grass.osgeo.org/grass8/manuals/addons/>
 
-The redirect to the latest grass7x directory is defined on grass.osgeo.org:
-  /etc/apache2/includes/grass.osgeo.org.inc
+The redirect to the latest grassX directory is defined on grass.osgeo.org:
+/etc/apache2/includes/grass.osgeo.org.inc
 
 Procedure building of binaries (Windows):
 
-* Addons module are compiled on build server, currently at the CTU in Prague) and
+- Addons module are compiled on winGRASS build server, at the CTU in Prague) and
   publishing their manual pages on publishing server, i.e. grass.osgeo.org.
-* A new compilation is triggered every time when a commit is done in the Addons-SVN.
-* Logs:
-  * Linux log files:   <https://grass.osgeo.org/addons/grass7/logs> (compiled on
+- A new compilation is triggered every time a commit is done in the Addons repo.
+- Logs:
+  - Linux log files: <https://grass.osgeo.org/addons/grass8/logs> (compiled on
     `grasslxd` on `osgeo7`)
-  * Windows log files: <http://wingrass.fsv.cvut.cz/grass78/x86_64/addons/latest/logs/>
+  - Windows log files: <http://wingrass.fsv.cvut.cz/grass8X/addons/grass-XXXdev/logs/>
 
 Procedure of granting write access to Addons repo:
 
-* Request procedure: <https://trac.osgeo.org/grass/wiki/HowToContribute#WriteaccesstotheGRASS-Addons-SVNrepository>
-* Adding OSGeo-ID:   <https://www.osgeo.org/cgi-bin/auth/ldap_group.py?group=grass_addons>
-* Adding contributor: <https://trac.osgeo.org/grass/browser/grass-addons/contributors.csv>
-  (via SVN commit)
-* Confirm request in grass-psc and give instructions concerning code style etc
+- Request procedure: <https://github.com/OSGeo/grass/blob/main/doc/development/submitting/submitting.md>
+- Adding OSGeo-ID: <https://www.osgeo.org/community/getting-started-osgeo/osgeo_userid/>
+- Adding contributor: <https://trac.osgeo.org/grass/browser/grass-addons/contributors.csv>
+  (via git commit)
+- Confirm request in grass-psc and give instructions concerning code style etc
   (see archive for examples)
 
-XML file for g.extension: <https://grass.osgeo.org/addons/grass7/modules.xml>
+XML file for g.extension: <https://grass.osgeo.org/addons/grass8/modules.xml>
 
-* generated in grass-addons/utils/addons/grass-addons-publish.sh
+- generated in grass-addons/utils/addons/grass-addons-publish.sh
 
 ## GRASS Travis CI
 
 Maintainer: Martin Landa
 
-* <https://travis-ci.org/GRASS-GIS>
-* <https://github.com/OSGeo/grass>
-* OLD: <https://github.com/GRASS-GIS/grass-ci>
-* <https://github.com/OSGeo/grass-addons/tree/master/utils/grass-ci/>
+- <https://travis-ci.org/GRASS-GIS>
+- <https://github.com/OSGeo/grass>
+- OLD: <https://github.com/GRASS-GIS/grass-ci>
+- <https://github.com/OSGeo/grass-addons/tree/master/utils/grass-ci/>
 
 Travis CI control files:
- trunk/.travis/
-   linux.before_install.sh
-   linux.install.sh
-   linux.script.sh
+
+```bash
+~/.travis/
+  linux.before_install.sh
+  linux.install.sh
+  linux.script.sh
+```
```

Comment:
I don't think this is needed. It is standardized location. The code block is now unclear anyway.

Reply:
Or not? Maybe just mention the dir name like for GitHub Actions.